mujoco_warp/_src/solver.py

```diff
@@ -2452,13 +2452,7 @@ def solve_done(
   efc_done_out[worldid] = (improvement < opt_tolerance) or (gradient < opt_tolerance)
 
 
-@event_scope
-def solve(m: types.Model, d: types.Data):
-  """Finds forces that satisfy constraints."""
-
-  # warmstart
-  wp.copy(d.qacc, d.qacc_warmstart)
-
+def solve_context(m: types.Model, d: types.Data, grad: bool = True):
```

Comment:
should this be called `create_context`?

Reply:
changed `solve_context` to `create_context`